verification/curator-service/api/src/model/source.ts

```diff
@@ -31,31 +31,46 @@ const fieldSchema = new mongoose.Schema({
     },
 });
 
-interface Parsing {
+interface RegexParsing {
     fields: Array<Field>;
 }
 
-const parsingSchema = new mongoose.Schema({
+const regexParsingSchema = new mongoose.Schema({
     fields: [fieldSchema],
 });
 
+interface Parser {
+    awsLambdaArn: string;
+}
+
+const parserSchema = new mongoose.Schema({
+    awsLambdaArn: {
+        type: String,
+        required: 'Enter a parser AWS Lambda ARN',
+    },
+});
+
+interface Schedule {
+    awsRuleArn: string;
+}
+
+const scheduleSchema = new mongoose.Schema({
+    awsRuleArn: {
```

Comment:
same here -- planning to validate format (e.g. arn:aws:cronwatch prefix)?

Reply:
Yep, per above.